src/languageservice/services/yamlCodeActions.ts

```diff
@@ -312,9 +316,52 @@ export class YamlCodeActions {
     }
     return results;
   }
+
+  /**
+   * Check if diagnostic contains info for quick fix
+   * Supports Enum/Const/Property mismatch
+   */
+  private getPossibleQuickFixValues(diagnostic: Diagnostic): string[] | undefined {
+    if (typeof diagnostic.data !== 'object') {
+      return;
+    }
+    if (
+      diagnostic.code === ErrorCode.EnumValueMismatch &&
+      'values' in diagnostic.data &&
+      Array.isArray((diagnostic.data as YamlDiagnosticData).values)
+    ) {
+      return (diagnostic.data as YamlDiagnosticData).values;
+    } else if (
+      diagnostic.code === ErrorCode.PropertyExpected &&
+      'properties' in diagnostic.data &&
+      Array.isArray((diagnostic.data as YamlDiagnosticData).properties)
+    ) {
+      return (diagnostic.data as YamlDiagnosticData).properties;
+    }
+  }
+
+  private getQuickFixForPropertyOrValueMismatch(diagnostics: Diagnostic[], document: TextDocument): CodeAction[] {
+    const results: CodeAction[] = [];
+    for (const diagnostic of diagnostics) {
+      const values = this.getPossibleQuickFixValues(diagnostic);
+      if (!values?.length) {
+        continue;
+      }
+      for (const value of values) {
+        results.push(
+          CodeAction.create(
+            value,
```

Comment:
This returns only the value which is not very descriptive of the operation. perhaps we can replace it with something like 
```suggestion
            `Use value "${value}"`,
```

Reply:
I don't have preferences here, I can remember that I tried a few combinations but I ended up with only `{value}`.
I was inspired by the English spelling checker extension (Code Spell Checker extension)
![image](https://github.com/redhat-developer/yaml-language-server/assets/38421337/34b2b548-b42d-42a1-9bd2-1310f0ba6ffd)

small disadvantage of `Use value "${value}"` is that this PR adds quickfix also for properties of the object (not value), so we need to display the text based on the error type
 - but for sure it's not a problem
 
 so please make a decision